### PR TITLE
Create 2020-05-12-do-you-speak-prestashop-may-2020.md

### PR DESCRIPTION
**Pull request by Junebyun**
Create 2020-05-12-do-you-speak-prestashop-may-2020.md

### CODE REVIEW

File: news/_posts/translation/2020-05-12-do-you-speak-prestashop-may-2020.md [hunk 1]
```diff
@@ -0,0 +1,131 @@
...
+It might be inconvenient for now but PrestaShop's developers team have in the backlog projects to make all new translations available for all shops from the 1.7.5 version and on, so let’s keep a positive mind! :sparkles:
+
+### :two_hearts: Special thanks to newcomers
+
+Many people subscribe to the project(s) of their choice each month, but not so many start translating straightaway. For that reason, the PrestaShop team would like to thank the ones who rushed into the effort immediately! So lots of love to the dedicated: choefer, Çocuk Saatleri (cocuksaatleri), nenowasaleh, Pacho Gonzalez (Pachogzz), Puunaama1 and Stella Georgakopoulou (Stella_Geo). See you soon on Crowdin… and in PrestaShop :stuck_out_tongue_winking_eye:
+
+
+## A few stats
+
+* **37 members** were active on the project this month.
```
Comment (LouiseBonnard) [inline news/_posts/translation/2020-05-12-do-you-speak-prestashop-may-2020.md:40]: And 38 active members.

File: news/_posts/translation/2020-05-12-do-you-speak-prestashop-may-2020.md [hunk 2]
```diff
@@ -0,0 +1,131 @@
...
+ 8. | Zoran Tejic (zotamal) | Serbian (Latin) | 588
+ 9. | nenowasaleh | Swahili | 568
+10. | Stella Georgakopoulou (Stella_Geo) | Greek | 500
+ 
+ 
+#### Top 10 proofreaders in May:
+ 
+| | Proofreader | Language | # Words
+|-| ---------- | -------- | ----------------
+1. | Rijad Osmanovic (rijado) | Bosnian | 40,220
```
Comment (LouiseBonnard) [inline news/_posts/translation/2020-05-12-do-you-speak-prestashop-may-2020.md]: Oops, it looks like there is a space missing at the beginning.

File: news/_posts/translation/2020-05-12-do-you-speak-prestashop-may-2020.md [hunk 3]
```diff
@@ -0,0 +1,131 @@
...
+Remember, you can see who's been contributing to our translation project thanks to the [Translators page](http://translators.prestashop.com/).
+ 
+ 
+## Complete translations
+ 
+#### Fully translated languages
+ 
+At the end of May 2020, PrestaShop 1.7.7 was fully available (= 100% translated and validated) in **11 languages**:
+ 
+| Danish | **Spanish** | **Italian** | **Dutch** | Portuguese, Brazil | Slovene | **Sweedish** | Chinese Traditional | **Greek** | **Slovak** | **French**|
```
Comment (LouiseBonnard) [inline news/_posts/translation/2020-05-12-do-you-speak-prestashop-may-2020.md]: Watch out, there's a typo in the _Swedish_ word. Also, it's better to add them in alphabetical order.

File: news/_posts/translation/2020-05-12-do-you-speak-prestashop-may-2020.md [hunk 4]
```diff
@@ -0,0 +1,131 @@
...
+ 
+ 
+## Languages with the best evolution
+ 
+In comparison with April 2020, the following languages had the best progress thanks to the translation community :sparkles:
+ 
+* **Bosnian** (+27% to reach 58% approved)
+* **Esperanto** (+16% to reach 25% translated)
+* **Catalan** (+9% to reach 100% translated)
```
Comment (LouiseBonnard) [inline news/_posts/translation/2020-05-12-do-you-speak-prestashop-may-2020.md:103]: 🎉🎉🎉

File: news/_posts/translation/2020-05-12-do-you-speak-prestashop-may-2020.md [hunk 5]
```diff
@@ -0,0 +1,131 @@
...
+* **Spanish, Venezuela** (52% vs 11%).
+* **Galician** (100% vs 59%).
+* **Esperanto** (25% vs 0%).
+* **Tagalog** (48% vs 24%).
+ 
+![Languages that need proofreading](/assets/images/2020/06/Build-Crowdin-proofreading-May20.png)
+ 
+If you wish to help to proofread what has been translated, please contact PrestaShop with the language you'd like to proofread: just send an email to translation@prestashop.com. Your help is needed!
+ 
+If you haven't joined us on Crowdin yet, [it's never too late](https://crowdin.com/project/prestashop-official) :wink:
```
Comment (LouiseBonnard) [inline news/_posts/translation/2020-05-12-do-you-speak-prestashop-may-2020.md]:
```suggestion
If you haven't joined us on Crowdin yet, [it's never too late](https://crowdin.com/project/prestashop-official)! :wink:
```

File: news/_posts/translation/2020-05-12-do-you-speak-prestashop-may-2020.md [hunk 6]
```diff
@@ -0,0 +1,131 @@
...
+ 
+If you want to gather your fellow translators to work towards a better harmonization, start a glossary, or anything else, do let me know: I'll include a word about it in the next monthly report.
+ 
+Do you have a question, a remark? Don't hesitate to leave a comment. See you next month :raising_hand:
```
Comment (LouiseBonnard) [inline news/_posts/translation/2020-05-12-do-you-speak-prestashop-may-2020.md]:
```suggestion
Do you have a question, a remark? Don't hesitate to leave a comment. See you next month! :raising_hand:
```

File: news/_posts/translation/2020-05-12-do-you-speak-prestashop-may-2020.md [hunk 7]
```diff
@@ -0,0 +1,131 @@
...
+![Best translation progress for May 2020](/assets/images/2020/06/Build-Crowdin-progress-May20.png)
+ 
+A huge thank you to all the contributors!
+ 
+Of course, this is highlighting the languages that made some progress with new translations; but it doesn't mean that the languages that aren't mentioned here aren't active. Indeed, some editing and rewriting could be going on, but the percentage of translation wouldn't be modified (since it's working on strings that are already translated). So let's not forget about the work of other proofreaders! Thanks to you too!
+ 
+ 
+## Languages that need (more) proofreaders
+ 
+A translated string will not be available in PrestaShop as long as it is not validated. For this reason, it’s important to keep a high level of validated strings vs. translated strings, to make sure everyone benefits from the latest translations :fire:
```
Comment (LouiseBonnard) [inline news/_posts/translation/2020-05-12-do-you-speak-prestashop-may-2020.md]:
```suggestion
A translated string will not be available in PrestaShop as long as it is not validated. For this reason, it’s important to keep a high level of validated strings vs. translated strings, to make sure everyone benefits from the latest translations! :fire:
```

File: news/_posts/translation/2020-05-12-do-you-speak-prestashop-may-2020.md [hunk 8]
```diff
@@ -0,0 +1,131 @@
...
+#### Fully translated languages
+ 
+At the end of May 2020, PrestaShop 1.7.7 was fully available (= 100% translated and validated) in **11 languages**:
+ 
+| Danish | **Spanish** | **Italian** | **Dutch** | Portuguese, Brazil | Slovene | **Sweedish** | Chinese Traditional | **Greek** | **Slovak** | **French**|
+ 
+ 
+## Languages with the best evolution
+ 
+In comparison with April 2020, the following languages had the best progress thanks to the translation community :sparkles:
```
Comment (LouiseBonnard) [inline news/_posts/translation/2020-05-12-do-you-speak-prestashop-may-2020.md]:
```suggestion
In comparison with April 2020, the following languages had the best progress thanks to the translation community:
```

**Comment by ttoine** [comment]
do you need to add pictures ?

File: news/_posts/translation/2020-05-12-do-you-speak-prestashop-may-2020.md [hunk 9]
```diff
@@ -0,0 +1,131 @@
...
+## Project news
+
+
+### First news, Swahili will be available in PrestaShop 1.7.8 :tada:
+
+I have the pleasure to announce to you that PrestaShop will finally be available in Swahili in the 1.7.8 version! I thus encourage you to translate, suggest and proofread the PrestaShop Swahili 1.7.7 translation project, as that will make it available to you and worldwide users as well. Currently, there aren’t many Swahili translators on PrestaShop’s Crowdin community, so if you know any Swahili natives, spread the word and do not hesitate to invite them to participate in the [PrestaShop Swahili translations project](https://crowdin.com/project/prestashop-official/sw#), they will be more than welcome! :heart_eyes:
+
+### Daily language packs updates
+
+Many of you asked why the translated strings were not updated in their shop, even though they have translated and approved strings, so I would like to take this opportunity to explain to you how translation packs are updated in PrestaShop now.
```
[the remark this answers is not in the document]
Comment (Junebyun) [reply news/_posts/translation/2020-05-12-do-you-speak-prestashop-may-2020.md]: The _many of you_ is justified but I think saying it in a more positive way will be better!

File: news/_posts/translation/2020-05-12-do-you-speak-prestashop-may-2020.md [hunk 10]
```diff
@@ -0,0 +1,131 @@
...
+ 3. | SeongHyeon Cho (jaymz9634) | Korean | 832
+ 4. | Gerardas (gerardas) | Lithuanian| 635
+ 5. | Zoran Tejic (zotamal) | Serbian (Latin) | 588
+ 6. | David Gonzalez (davidglezz) | Spanish | 537
+ 7. | Stamatis (breezer) | Greek | 512
+ 8. | Patrik Lindblom (extendus) | Swedish | 489
+ 9. | webdvl (megashopba) | Slovak; Czech | 455
+10. | Gabriel Tenita (ggedamed) | Romanian | 438
+ 
+Congrats, and welcome to our new top contributors :clap:
```
[the remark this answers is not in the document]
Comment (Junebyun) [reply news/_posts/translation/2020-05-12-do-you-speak-prestashop-may-2020.md]: I'll make sure to check the punctuation after every emoji next time haha!

File: news/_posts/translation/2020-05-12-do-you-speak-prestashop-may-2020.md [hunk 11]
```diff
@@ -0,0 +1,131 @@
...
+
+### :two_hearts: Special thanks to newcomers
+
+Many people subscribe to the project(s) of their choice each month, but not so many start translating straightaway. For that reason, the PrestaShop team would like to thank the ones who rushed into the effort immediately! So lots of love to the dedicated: choefer, Çocuk Saatleri (cocuksaatleri), nenowasaleh, Pacho Gonzalez (Pachogzz), Puunaama1 and Stella Georgakopoulou (Stella_Geo). See you soon on Crowdin… and in PrestaShop :stuck_out_tongue_winking_eye:
+
+
+## A few stats
+
+* **37 members** were active on the project this month.
+* A total of **53,384 words have been translated** and **46,675 validated**.
```
[the remark this answers is not in the document]
Comment (Junebyun) [reply news/_posts/translation/2020-05-12-do-you-speak-prestashop-may-2020.md:41]: Did delete _Nicolas Etienne (nicolas.etienne)_? He translated 12 Spanish, Mexico words! If you delete him from the list it's 53,384 translated words

File: news/_posts/translation/2020-05-12-do-you-speak-prestashop-may-2020.md (hunk 1 above)
Comment (Junebyun) [reply news/_posts/translation/2020-05-12-do-you-speak-prestashop-may-2020.md:40]: So 37 members without Nicolas Etienne :)

File: news/_posts/translation/2020-05-12-do-you-speak-prestashop-may-2020.md (hunk 2 above)
Comment (ttoine) [reply news/_posts/translation/2020-05-12-do-you-speak-prestashop-may-2020.md]: can you explain where the space is missing ? can you provide a suggestion ?

File: news/_posts/translation/2020-05-12-do-you-speak-prestashop-may-2020.md [hunk 12]
```diff
@@ -0,0 +1,132 @@
...
+
+### Daily language packs updates
+
+I would like to take this opportunity to explain to you how translation packs are updated in PrestaShop now, after you translate and approve strings.
+
+Translations are updated everyday with a 24 hours delay. Every morning our developers send an export request to Crowdin, but since the exporting process takes some time, they download the export made the day before and send the files to [Github](https://github.com/PrestaShop/TranslationFiles). When the repository is updated, a robot will gather the translation files and make them available to download. 
+But one detail to take into account is that only translations of the last version, the 1.7.7, are updated. 
+
+Since PrestaShop 1.7.7 hasn’t been officially released yet, what you can do is to translate and approve strings of the 1.7.7 version, retrieve these strings from translationfiles and upload them on: app/Resources/Translations/{localeCode}. 
+It might be inconvenient for now but PrestaShop's developers team have in the backlog projects to make all new translations available for all shops from the 1.7.5 version and on, so let’s keep a positive mind! :sparkles:
```
Comment (Junebyun) [inline news/_posts/translation/2020-05-12-do-you-speak-prestashop-may-2020.md]: @eternoendless Could you check if the information is correct? Thank you :)

File: news/_posts/translation/2020-05-12-do-you-speak-prestashop-may-2020.md [hunk 13]
```diff
@@ -0,0 +1,132 @@
...
+Contributing to PrestaShop is not only about the code, it's also about taking part in the [PrestaShop translation project](https://crowdin.com/project/prestashop-official)! This report will tell you how the translations of the software evolved in May.
+
+![Crowdin Monthly banner](/assets/images/2019/01/Build-Crowdin-banner.jpg)
+
+## Project news
+
+
+### First news, Swahili will be available in PrestaShop 1.7.8! :tada:
+
+I have the pleasure to announce that PrestaShop will finally be available in Swahili in the 1.7.8 version! I thus encourage you to translate, suggest, and proofread the PrestaShop Swahili 1.7.7 translation project. Currently, there aren’t many Swahili translators on PrestaShop’s Crowdin community, so if you know any Swahili natives, spread the word and do not hesitate to invite them to participate in the [PrestaShop Swahili translations project](https://crowdin.com/project/prestashop-official/sw#), they will be more than welcome! :heart_eyes:
```
Comment (eternoendless) [inline news/_posts/translation/2020-05-12-do-you-speak-prestashop-may-2020.md]: I find this a little confusing:

> PrestaShop will finally be available in Swahili in **the 1.7.8 version**!
> I thus encourage you to translate, (...) the PrestaShop Swahili **1.7.7 translation project**

Comment (Junebyun) [reply news/_posts/translation/2020-05-12-do-you-speak-prestashop-may-2020.md]: @eternoendless thanks for the comment! I'll simply take off 1.7.7 😉

**Pull request merged**